example/index.php

```diff
@@ -1,21 +1,43 @@
 <?php
 require_once __DIR__ . '/../src/boot.php';
```

Comment:
`require_once __DIR__ . '/../src/boot.php';` can be run when not using composer autoloader, comment one of those lines and mention this in a comment. `require_once './vendor/autoload.php';` uses composer.